plugins/services/src/js/pages/task-details/TaskLogsTab.js

```diff
@@ -234,22 +234,21 @@ class TaskLogsTab extends mixin(StoreMixin) {
   getDownloadButton() {
     const {task} = this.props;
     const {selectedStream} = this.state;
-    const params = getLogParameters(task, {filter: {STREAM: selectedStream}});
+    const params = getLogParameters(task, {
+      postfix: selectedStream.toLowerCase(), // This will be added to the name
```

Comment:
Yup, that's the problem. Thanks for noticing. That's what you get for making a minor change right at the end and don't test it through :man_shrugging: